beetsplug/mbcollection.py

```diff
@@ -28,8 +28,6 @@
 
 SUBMISSION_CHUNK_SIZE = 200
 UUID_REGEX = r'^[a-f0-9]{8}(-[a-f0-9]{4}){3}-[a-f0-9]{12}$'
-AUTH_ERRORS = (TokenRequestDenied, TokenMissing, VerifierMissing)
-
```

Comment:
The right thing to do here is probably to import these exception names so we can use them, like this: https://github.com/beetbox/beets/blob/master/beetsplug/beatport.py#L26-L27